example/rocketmq_v1alpha1_broker_cr.yaml

```diff
@@ -28,6 +28,39 @@ data:
     brokerRole=ASYNC_MASTER
 
 ---
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: broker-acl-config
+data:
+  plain_acl.yml: |
+    globalWhiteRemoteAddresses:
+      - 10.10.103.*
+      - 192.168.0.*
+
+    accounts:
+      - accessKey: RocketMQ
+        secretKey: 123456789
```

Comment:
Is any more secure ways to store `secretKey`? Exposing it in confimap may not be a secure practice, but currently I couldn't find an appropriate way.

Reply:
Can we store the whole plain_acl.yml in `Secret`?